server/services/datacommons.py

```diff
@@ -193,8 +192,13 @@ def get_property_values(dcids,
     return results
 
 
-def get_triples(dcids, limit=_MAX_LIMIT):
-    # Generate the GetTriple query and send the request.
+def get_triples(dcids, limit=0):
```

Comment:
This API is used by pv tree tool, hence adding a new one "get_triples_raw"

Reply:
We should update the comment so we can tell these apart in the future. Perhaps this should be renamed instead, while the other should remain as get_triples.